kraft.yaml

```diff
@@ -36,6 +32,8 @@ libraries:
       - CONFIG_LIBPYTHON3_EXTENSION_UUID=y
       - CONFIG_LIBPYTHON3_EXTENSION_ZLIB=y
       - CONFIG_LIBPYTHON3_MAIN_FUNCTION=y
+  libuuid:
+    version: stable
```

Comment:
```suggestion
  libuuid:
    version: staging
```

Stable currently has a breaking bug

Reply:
@craciunoiuc is the solution to use `staging` or to tag `libuuid` properly?
If we are all ok with using `staging` I will change everything from `stable` to `staging`, but I think using `stable` is a good idea if all repositories are tagged properly.

Reply:
Yes, it's better to tag properly

That library was forgotten about unfortunately, so it needs to be brought up to date

Reply:
@craciunoiuc @razvand so what should we do here, use staging or tag `libuuid`

Reply:
we do what milord orders

@razvand

Reply:
@craciunoiuc , the [`stable` branch](https://github.com/unikraft/lib-libuuid/tree/stable) is now identical to the [`staging` branch](https://github.com/unikraft/lib-libuuid/tree/staging).

Let's get this in.